src/app/control-planes/views/ControlPlaneDetailView.vue

```diff
@@ -13,111 +13,166 @@
+                          <div
+                            v-if="can('create zones') && (zoneOverviewsData?.items?.length ?? 0 > 0)"
+                            class="card-actions"
+                          >
+                            <KButton
+                              appearance="primary"
+                              :to="{ name: 'zone-create-view' }"
+                            >
+                              <AddIcon :size="KUI_ICON_SIZE_30" />
+
+                              {{ t('zones.index.create') }}
+                            </KButton>
+                          </div>
+                        </div>
+
+                        <ZoneControlPlanesList
+                          data-testid="zone-control-planes-details"
+                          :items="zoneOverviewsData?.items"
+                        />
+                      </template>
+                    </template>
+                  </KCard>
+
+                  <KCard>
+                    <template #body>
+                      <ErrorBlock
+                        v-if="meshInsightsError"
+                        :error="meshInsightsError"
+                      />
+
+                      <template v-else>
+                        <div class="card-header">
+                          <div class="card-title">
+                            <h2>{{ t('main-overview.detail.meshes.title') }}</h2>
+
+                            <RouterLink :to="{ name: 'mesh-list-view' }">
+                              {{ t('main-overview.detail.health.view_all') }}
+                            </RouterLink>
+                          </div>
+                        </div>
+
+                        <MeshInsightsList
+                          data-testid="meshes-details"
+                          :items="meshInsightsData?.items"
+                        />
+                      </template>
+                    </template>
+                  </KCard>
+                </div>
+              </div>
             </template>
-          </KCard>
-        </div>
-      </div>
+          </DataSource>
+        </DataSource>
+      </DataSource>
     </AppView>
   </RouteView>
 </template>
 
 <script lang="ts" setup>
 import { KUI_ICON_SIZE_30 } from '@kong/design-tokens'
 import { AddIcon } from '@kong/icons'
+import { ref } from 'vue'
 
+import { GlobalInsightSource } from '../sources'
 import ErrorBlock from '@/app/common/ErrorBlock.vue'
+import LoadingBlock from '@/app/common/LoadingBlock.vue'
 import { useControlPlaneStatus } from '@/app/control-planes'
 import MeshInsightsList from '@/app/meshes/components/MeshInsightsList.vue'
 import type { MeshInsightCollectionSource } from '@/app/meshes/sources'
 import ZoneControlPlanesList from '@/app/zones/components/ZoneControlPlanesList.vue'
 import type { ZoneOverviewCollectionSource } from '@/app/zones/sources'
 
 const ControlPlaneStatus = useControlPlaneStatus()
+
+const isShowingOnboardingAlert = ref(true)
```

Comment:
Now that this in in the View and not the never re-rendered ApplicationShell, I think this would mean that after the pressing the cross to close the alert and coming back to the homepage again, the alert would remain?

Reply:
Yes, but the same happens when reloading the page right now. We don’t really attempt to persist this decision anymore. We should probably do so using localStorage.

Reply:
I agree, but in the meantime but at the very least right now, at least we don't continue showing the thing everytime you leave the homepage to say go to the onboarding and come back again.

I suppose this was why I was asking what the aim of this PR is? If we aren't adding the localStorage then this takes away functionality rather than add. Could you update the description of the PR here with the aim of it? I think then it will be easier to understand whats going on here.

Reply:
I’ve added the local storage mechanism now. I also updated the PR description to reflect the current state of this PR more closely.